db_service/vcap_test.go

```diff
@@ -0,0 +1,287 @@
NhTmlFV0VKVGlcblduNDRva01qUUlZNGRQdGcvaHpjUit5cWxwWGg4a3p2czVITlI5MXg3eDJJbmlTeUpnK2lEQnlKSlZBdGowd3JcbjFKZlgzVTdpOFJLdThGTENGSkV4OEZZREd3eWRwNlZUYjkyL3RjR3NQSDl0NXVGSVc3TjUvTk5zZXVYaFBxSzNcbkgrdUQ4TkRGQWdNQkFBRUNnZ0VBSlVzNUxiNWdyUTNYQlIyZWxZbTJaZzcxV2FtTUxOcVR0b0kzYXp3V1VDbStcbllMRzJTSjlmRXlBRTU2a0hDWk0wYis1am9NVkFlSG1Va21QMHhHUUNzM2pJVzhuZGRBZjVGL0xMYXBibXZIdndcbnNZdXlKbXlkbVpSYjgrVEI2aWlmaElRVVRKVEIwd2l1OUlSQkk0YUdGa3Z2UE94aG9sblVWK3htcEFtUXMyd1pcblYxcEZSOTBtYVFWbEFLMVpGNlZWUmZTMnl4RzVtc1N4dUxkekpidjdadURYN2ZGL1NtMXlKbURjMFVzalVNWEJcbnp0WGU5dXFXZkE5RCsyamtGb1d3MGZjVnZPSE4zbDV3TFNpTTI3cXNNWksxL0YwemUwKzkwVUpyd0xnNGlxQ2dcbmxXVHFnR3FZaktOUHp6TFdJU0ZNMnpaek5yNk1Ha2dwMEIyYmhwWWVBd0tCZ1FEcEFnVUpHUGQwc0ZkYXhGVWVcbk1PbkVRZjNrYW5PRUY1eGR4UXNxdUpGNWhBMjE3YjFVQllCRjM1cGJKOURkMGsrUjdVV3dqL0Q2OUtiV3NscS9cbkFEamltY2NlalZ4V1ZUM1BTWnhySVpYRE1NbjR5eHFIaW1EaDdUSDZPa3RINFJZbU9zN2JqcEcxdzEvYnFvTlhcbjJCM1pOOEI2THRsZjdqSHdjbUREVHRqR2R3S0JnUURueC8yZ0F0ZVcxNkxaTnp1aWU3WFJjeWd2NGRZUXdoUlpcbmR6SHJLc3EvMjdjeTAwUE9wWk9BK0FtVjFwdVc1V1Y4OENtVmJsQzQwckR2cnlUaHhPODJWN0toZ3BMRWVsaUdcblMzQTQ1ell3eGluT2dYMTZKKzVaSmFYL0xLMTlyV1V5L3I0b0FMOUR4cEl2dVFpSkwyRHVZM0NzeUp6YWF0bjFcbjQ1UW5heTdsb3dLQmdFSndaLzByR0V3MmlBSUNuMzZuVmRDM1BHem9DWjR0bVZHSGdPS2lsQ0NCRGVQRk1Vb0dcbjg0ZDQ5YXR1VS9rY0ljSXJWTWErbEdrS1g1UXljUHVyVlkwUGFoNkZFa0l2dGhzb0V5amMvN1lUY0ZPM25nM3RcbjRDZ3JtU2VQZmEyMk9ibVc1U3Jub1JhaDZmQlowMisxMlBUNkY3RC9NTTVRdmY2Z3JvU2lNOStMQW9HQkFNVndcbnZHRlk2bk9KWHlTd0F6SEhOanVrWUNCaHZhdHEyRkRaMDRFalk3RUpwa1k2WnpHYUpFdWhmdkRQN3B3YzcxWDlcbmN6N2l5UXFZRjdjbE9FTEdNb3ZWS3NxZ1l3dlJ1S1Uxai9RNUtSVmxTT21ycnNxblIwZFRaZE00S05XOUprN0pcbmFBekZqaWhhOTk2RlBYczNDOWdtaHkzNGVuMG90bURhcXpMay8vOEhBb0dCQUtSQW5xeXpud08rVUZBQjBkSWFcbk1VYUdza0hJUmNvNmo4d2FqNkRlSzJqdThYRnZlZGxuOVExV2U3djRXRHBxc1BCWWhYMFdoc0ViM2pheGtRT2xcbjB4ZHprRDMrYUpBSjNFSlBpcEhFeDZscVM1aFRGdlBVM0RMTmFSbDhmQVZCdGREWVJNRmw2Y0xlMDF5b1h3UkRcbkJyU2drbHVTSU1yb1JxOGMwZVhseU53ZFxuLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLVxuIiwKICAiY2xpZW50X2VtYWlsIjogInBjZi1iaW5kaW5nLXRlc3RiaW5kQGpsZXdpc2lpaS1nc2IuaWFtLmdzZXJ2aWNlYWNjb3VudC5jb20iLAogICJjbGllbnRfaWQiOiAiMTA4ODY4NDM0NDUwOTcyMDgyNjYzIiwKICAiYXV0aF91cmkiOiAiaHR0cHM6Ly9hY2NvdW50cy5nb29nbGUuY29tL28vb2F1dGgyL2F1dGgiLAogICJ0b2tlbl91cmkiOiAiaHR0cHM6Ly9vYXV0aDIuZ29vZ2xlYXBpcy5jb20vdG9rZW4iLAogICJhdXRoX3Byb3ZpZGVyX3g1MDlfY2VydF91cmwiOiAiaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vb2F1dGgyL3YxL2NlcnRzIiwKICAiY2xpZW50X3g1MDlfY2VydF91cmwiOiAiaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vcm9ib3QvdjEvbWV0YWRhdGEveDUwOS9wY2YtYmluZGluZy10ZXN0YmluZCU0MGpsZXdpc2lpaS1nc2IuaWFtLmdzZXJ2aWNlYWNjb3VudC5jb20iCn0K", "ProjectId": "jlewisiii-gsb", "Sha1Fingerprint": "aa3bade266136f733642ebdb4992b89eb05f83c4", "UniqueId": "108868434450972082663", "UriPrefix": "", "Username": "newuseraccount", "database_name": "service_broker", "host": "104.154.90.3", "instance_name": "pcf-sb-1-1561406852899716453", "last_master_operation_id": "", "region": "", "uri": "mysql://newuseraccount:Ukd7QEmrfC7xMRqNmTzHCbNnmBtNceys1olOzLoSm4k@104.154.90.3/service_broker?ssl_mode=required"}},
```

Comment:
I think it would help to populate these with dummy or truncated values for the keys and certs.

Possibly also remove references to the project name: `jlewisiii-gsb`